src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java

```diff
@@ -604,6 +621,13 @@ public void onCompletion(client_operator clientOP) {
       return promise;
     }
 
+    try {
```

Comment:
```
promise.setFailure(new PException("Exceed write limit threshold:" + e.getMessage()));
```
This line repeats for every operation. Wrap it into a util:

```
  private void limitWrite(Promise promise, IllegalArgumentException exp) {
    if(exp != null) {
      promise.setFailure(new PException("Exceed write limit threshold:" + exp.getMessage()));
    }
  }
```

Reply:
No need to return `Exception`, only wrap it like `handleReplicaException`:
https://github.com/XiaoMi/pegasus-java-client/blob/5d57146f73a0d2d014cdf23c5d39aedb35f263ba/src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java#L1755-L1756

Reply:
```
    try {
      writeLimiter.validateMultiSet(hashKey, values);
    } catch (IllegalArgumentException e) {
      handleWriteLimiterException(promise, e.getMessage());
      return promise;
    }
```

compares to

```java
limitWrite(writeLimiter.validateMultiSet(hashKey, values));
```

See which one is better?

Reply:
`promise` need return if `writeLimiter.validateMultiSet(hashKey, values)` not return null, so the code seem should be:
```java
  IllegalArgumentException exp = writeLimiter.validateMultiSet(hashKey, values);
  limitWrite(promise, exp);
  if (exp != null) {
    return promise
  }
```
or
```java
  IllegalArgumentException exp = writeLimiter.validateMultiSet(hashKey, values);
  if(exp != null){
    limitWrite(promise, exp);
    return promise
  }
```
and java has try...cath, why return `exception` in one method and  judge the `Exception` whether `null` to handle the logic.